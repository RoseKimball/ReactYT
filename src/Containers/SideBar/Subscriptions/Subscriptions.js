import React, { Component } from 'react';
import Subscription from '../Subscription/Subscription';
import SideBarHeader from '../SideBarHeader/SideBarHeader';

export class Subscriptions extends Component {
    render() {
        return(
            <React.Fragment>
                <SideBarHeader title='Subscriptions'/>
                <Subscription label='ColleenBallinger' broadcasting/>
                <Subscription label='AwesomeScience' amountNewVideos={10}/>
                <Subscription label='Cut' amountNewVideos={3}/>
                <Subscription label='Jubilee' amountNewVideos={1}/>
            </React.Fragment>
        );
    };
};

export default Subscriptions;