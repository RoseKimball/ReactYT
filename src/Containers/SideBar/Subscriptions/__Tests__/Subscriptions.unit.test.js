import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, mount, render } from 'enzyme';
import Subscriptions from '../Subscriptions';

describe('Subscriptions', () => {
    test('renders Subscriptions', () => {
        const wrapper = shallow (
            <Subscriptions />
        )
        expect(wrapper).toMatchSnapshot();
    })
})