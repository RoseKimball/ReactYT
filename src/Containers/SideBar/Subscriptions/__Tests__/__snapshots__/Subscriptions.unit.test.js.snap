// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subscriptions renders Subscriptions 1`] = `ShallowWrapper {}`;
